kuma/javascript/src/header/main-menu.jsx

```diff
@@ -0,0 +1,229 @@
+                    },
+                    {
+                        url: `/${locale}/docs/Web/Guide`,
+                        label: gettext('Developer Guides')
+                    },
+                    {
+                        url: `/${locale}/docs/Web/Accessibility`,
+                        label: gettext('Accessibility')
+                    },
+                    {
+                        url: `/${locale}/docs/Games`,
+                        label: gettext('Game development')
+                    },
+                    {
+                        url: `/${locale}/docs/Web`,
+                        label: gettext('...more docs')
+                    }
+                ]
+            },
+            {
+                label: gettext('Feedback'),
+                items: [
+                    {
+                        url: `/${locale}/docs/MDN/Feedback`,
+                        label: gettext('Send Feedback')
+                    },
+                    {
+                        url: 'https://support.mozilla.org/',
+                        label: gettext('Get Firefox help'),
+                        external: true
+                    },
+                    {
+                        url: 'https://stackoverflow.com/',
+                        label: gettext('Get web development help'),
+                        external: true
+                    },
+                    {
+                        url: `/${locale}/docs/MDN/Community`,
+                        label: gettext('Join the MDN community')
+                    },
+                    {
+                        label: gettext('Report a content problem'),
+                        external: true,
+                        url:
+                            'https://github.com/mdn/sprints/issues/new?template=issue-template.md&projects=mdn/sprints/2&labels=user-report&title={{PATH}}'
+                    },
+                    {
+                        label: gettext('Report a bug'),
+                        external: true,
+                        url: 'https://bugzilla.mozilla.org/form.mdn'
+                    }
+                ]
+            }
+        ],
+        [locale]
+    );
+
+    /**
+     * Handles all interaction events
+     * @param {Object} event - The Event object
+     */
+    function interactionHandler(event: SyntheticEvent<HTMLButtonElement>) {
+        let mediaQuery = window.matchMedia('(min-width: 47.9375em)');
+        let currentTarget = event.target;
+
+        /* currentTarget instanceof 'HTMLButtonElement' is added
+           to keep Flow happy: https://github.com/facebook/flow/issues/218#issuecomment-74119319 */
+        if (
+            (!mediaQuery.matches,
+            event.type === 'touchstart' &&
+                currentTarget instanceof HTMLButtonElement &&
+                currentTarget.classList.contains('top-level-entry'))
+        ) {
+            if (currentTarget instanceof HTMLButtonElement) {
```

Comment:
Without this, Flow keeps inferring an error as it sees the variable as an `EventTarget` and not an HTML element. The fact that I check for this in the out conditional already seems to make no difference.

If I use `event.currentTarget` on line 127, it is happy but, that will always be the `nav` element that wraps the entire menu. So this is the only way I can keep flow happy.